todo_app/templates/index.html

```diff
@@ -8,25 +8,61 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
+    <div class="col-md-4">
+      <h2>To Do</h2>
       <ul class="list-group mb-4">
-        {% for item in items %}
+        {% for item in todo_items.items %}
```

Comment:
If we made the change suggested above to use a single view model, this will likely change to something like below (depending on variable names!)
```jinja
{% for item in todo_items.to_do_items %}
```